lib/engine/tile.rb

```diff
@@ -216,51 +216,53 @@ def inspect
       "<#{self.class.name}: #{name}, hex: #{@hex&.name}>"
     end
 
-    # returns hash where keys are cities, and values are the edge the city
-    # should be rendered at
-    def preferred_city_edges
+    # returns hash where keys are cities, and values are the edge the city or
+    # town should be rendered at
+    #
+    # "ct" for "city or town"
+    def preferred_city_town_edges
       # cache per rotation
-      @preferred_city_edges[@rotation] ||=
+      @preferred_city_town_edges[@rotation] ||=
```

Comment:
let's not cache by rotation i think this is error prone

instead clear the cache when rotated like the other stuff

Reply:
9488efb